tests/providers/amazon/aws/sensors/test_athena.py

```diff
@@ -36,26 +36,26 @@ def setUp(self):
             aws_conn_id='aws_default',
         )
 
-    @mock.patch.object(AWSAthenaHook, 'poll_query_status', side_effect=("SUCCEEDED",))
+    @mock.patch.object(AthenaHook, 'poll_query_status', side_effect=("SUCCEEDED",))
     def test_poke_success(self, mock_poll_query_status):
-        assert self.sensor.poke(None)
+        assert self.sensor.poke({})
 
-    @mock.patch.object(AWSAthenaHook, 'poll_query_status', side_effect=("RUNNING",))
+    @mock.patch.object(AthenaHook, 'poll_query_status', side_effect=("RUNNING",))
     def test_poke_running(self, mock_poll_query_status):
-        assert not self.sensor.poke(None)
+        assert not self.sensor.poke({})
 
-    @mock.patch.object(AWSAthenaHook, 'poll_query_status', side_effect=("QUEUED",))
+    @mock.patch.object(AthenaHook, 'poll_query_status', side_effect=("QUEUED",))
     def test_poke_queued(self, mock_poll_query_status):
-        assert not self.sensor.poke(None)
+        assert not self.sensor.poke({})
 
-    @mock.patch.object(AWSAthenaHook, 'poll_query_status', side_effect=("FAILED",))
+    @mock.patch.object(AthenaHook, 'poll_query_status', side_effect=("FAILED",))
     def test_poke_failed(self, mock_poll_query_status):
         with pytest.raises(AirflowException) as ctx:
-            self.sensor.poke(None)
+            self.sensor.poke({})
         assert 'Athena sensor failed' in str(ctx.value)
 
-    @mock.patch.object(AWSAthenaHook, 'poll_query_status', side_effect=("CANCELLED",))
+    @mock.patch.object(AthenaHook, 'poll_query_status', side_effect=("CANCELLED",))
     def test_poke_cancelled(self, mock_poll_query_status):
         with pytest.raises(AirflowException) as ctx:
-            self.sensor.poke(None)
+            self.sensor.poke({})
```

Comment:
This is not really related to the PR description?
I guess you fix something else here on the way? If so just add some description about it in the PR

Reply:
Yeah, sorry.   I'm really bad at that; I fix little things when I see them because I know I'll forget or won't get to it later.  IDE was throwing a type hint warning there, expecting a dict and got NoneType.  I'll add a comment to the description.